konami/iidx/product.md

```diff
@@ -0,0 +1,180 @@
+# Beatmania IIDX products
```

Comment:
My first thought would be to make this `konami/products/iidx.md` instead, and collecting the info about all IIDX stuff there until it gets too unwieldy for one file.

Reply:
You reviewed the file that is supposed to be removed, see my comment in line 3.